integration_test/third_party_apps_data/applications/hbase/enable

```diff
@@ -29,3 +29,4 @@ logging:
 EOF
 
 sudo service google-cloud-ops-agent restart
+sleep 60
```

Comment:
Is this left over from debugging? Or is this now required on Ubuntu 23.04?

Reply:
Not required. Just thought it was odd that every other `enable` script sleeps, and this one doesn't.